src/yunohost/app.py

```diff
@@ -457,18 +458,13 @@ def app_change_url(operation_logger, auth, app, domain, path):
     old_path = app_setting(app, "path")
 
     # Normalize path and domain format
-    domain = domain.strip().lower()
-
-    old_path = normalize_url_path(old_path)
-    path = normalize_url_path(path)
```

Comment:
You should really avoid merging several different modifications into one commit and prefer atomic commits instead, this make review harder.

Reply:
Sorry,

Will try to do better the next time

Reply:
Thx <3